content/Encapsulation/Encapsulation.objectives.md

```diff
@@ -0,0 +1,11 @@
+### Objectives for Encapsulation lesson:
+
+After completing this lesson, the students should be able to:
+
+ - Identify the basic principles and purpose of encapsulation in Object-Oriented Programming. 
+
+ - Use the `private` visibility keyword to encapsulate the properties and methods of a certain class.
```

Comment:
I think we need to put `and` in-between keyword and to

Reply:
the conjunction `and` is not required in this statement since we only present a single idea here. Please check. Thank you.